org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -428,12 +428,12 @@ public static LanguageServerWrapper getLSWrapperForConnection(@NonNull IProject
 		return getLSWrapper(project, serverDefinition);
 	}
 
-	private static LanguageServerWrapper getLSWrapper(@NonNull IProject project,
```

Comment:
this declaration was incorrect, as there is a check for null project on line 443